ui/ui-expenses/src/screens/list/list.reactions.js

```diff
@@ -1,9 +1,19 @@
-import pica from 'pica/dist/pica'
-
 export const load = (action, store, { http }) => {
   http('EXPENSES').get('/api/expenses')
 }
 
+export const submit = async ({ payload }, store, { pica }) => {
+  const img = await pica.resize(payload)
+  store.dispatch({ type: '@@ui/IMAGE_RESIZED', payload: img })
```

Comment:
the driver could dispatch this action I think.

`@@pica/IMAGE_RESIZED, payload: img`